addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1119,6 +1119,16 @@ const Wysiwyg = Widget.extend({
                     }
                     const html = $node.html();
                     for (const node of $nodes) {
+                        if (node.classList.contains('o_translation_without_style')) {
+                            // For generated elements such as the navigation
+                            // labels of website's table of content, only the
+                            // text of the referenced translation must be used.
+                            const text = $node.text();
+                            if (node.innerText !== text) {
+                                node.innerText = text;
+                            }
+                            continue;
+                        }
                         if (node.innerHTML !== html) {
                             node.innerHTML = html;
                         }
```

Comment:
@bso-odoo here, and when we use a styled title on the original language, the content will be updated from its nav item, I think it's a side effect of using the same id on both elements, can you check this :+1: 

https://user-images.githubusercontent.com/25032876/226689224-393389ab-d5ec-40af-ae41-4cd56a151b3c.mp4

Reply:
Now preventing the forward of the untyled copied values to "synonym" nodes.